.buildkite/integration.pipeline.yml

```diff
@@ -90,6 +107,23 @@ steps:
           instanceType: "c6g.2xlarge"
           image: "${IMAGE_UBUNTU_2204_ARM_64}"
 
+      - label: ":package: arm64: OTEL_COMPONENT tar.gz"
```

Comment:
`label: ":package: arm64: FIPS tar.gz"` and `":package: arm64: OTEL_COMPONENT tar.gz"` only differ in using the env variable `FIPS: "true"` or `OTEL_COMPONENT: "true"`, wonder whether using a [buildkite matrix](https://buildkite.com/docs/pipelines/configure/workflows/build-matrix) could help here?